www/lendingbot.js

```diff
@@ -53,11 +53,9 @@ function updateOutputCurrency(outputCurrency){
 // below zero accuracy will indicate precision after must significat digit
 // strips trailing zeros
 function prettyFloat(value, accuracy) {
-    var precision = Math.round(Math.log10(value));
-    precision = precision < 0 ? Math.abs(precision) + accuracy : accuracy;
-    var multiplier = Math.pow(10, precision);
-    var result = Math.round(value * multiplier) / multiplier;
-    return isNaN(result) ? "0" : parseFloat(result.toFixed(precision));
+  var precision = Math.round(Math.log10(value));
```

Comment:
@rnevet it would be great to keep same indention style in JS code (rest of the code uses 4 spaces).